### PR TITLE
Bug 1004734 - Create whatsnew-notification experiment. r=liuche, Bug 1253588 - Enable bookmark history menu experiment by default

### PR DESCRIPTION
**Pull request by leibovic**
@liuche Can you review this?

@milescrabill I tested this locally to confirm this experiment won't be active for any users, but I'd like for you to also confirm that this is how the bucket logic works. It's not clear to me whether the min/max are inclusive or exclusive. I'm assuming that this patch means that no buckets will be active, but it's possible it could also be active for users in bucket 0, if buckets are 0-indexed. I want us to update the README to clarify this.

### CODE REVIEW

**Comment by leibovic** [comment]
I confirmed this with this URL with UUID 1:
https://switchboard.stage.mozaws.net/v1?uuid=1&device=mako&lang=eng&country=USA&manufacturer=LGE&appId=org.mozilla.fennec&version=47.0a1

File: experiments.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "bookmark-history-menu": {
     "match": {
-      "appId": "^org.mozilla.fennec$"
+      "appId": "^org.mozilla.fennec*|^org.mozilla.firefox_beta$"
```
Comment (liuche) [inline experiments.json:4]: Do we want this on Nightly or Aurora too, or just beta and local builds?

File: experiments.json [hunk 2]
```diff
@@ -16,5 +16,13 @@
       "min": "0",
       "max": "100"
     }
+  },
+  "whatsnew-notification": {
+    "match": {
+    },
```
Comment (liuche) [inline experiments.json:22]: Nit: if match doesn't have any parameters, just have the open/close {} on the same line.

**Comment by liuche** [comment]
r+ from me

File: experiments.json (hunk 1 above)
Comment (leibovic) [inline experiments.json:4]: The `org.mozilla.fennec*` line matches Nightly (`org.mozilla.fennec`), Aurora (`org.mozilla.fennec_aurora`), and all local builds.

**Pull request merged**